components/mbedtls/port/net.c

```diff
@@ -201,10 +201,12 @@ int mbedtls_net_bind( mbedtls_net_context *ctx, const char *bind_ip, const char
  *
  * Note: on a blocking socket this function always returns 0!
  */
-static int net_would_block( const mbedtls_net_context *ctx )
+static int net_would_block( const mbedtls_net_context *ctx, int *errout )
 {
     int error = mbedtls_net_errno(ctx->fd);
 
+    if ( errout ) *errout = error;
```

Comment:
Minor style thing, but can you please put the assignment statement on a separate line inside curly braces. This in keeping with conditional style in the rest of the file.